Kickstarter-iOS/Views/Cells/DiscoveryPostcardCell.swift

```diff
@@ -320,6 +320,17 @@ internal final class DiscoveryPostcardCell: UITableViewCell, ValueCell {
   }
 
   @objc fileprivate func saveButtonTapped() {
+    if self.saveButton.isSelected {
```

Comment:
I think since we are using the same code for a UIButton to generate feedback in multiple places, we could extend `UIButton` and move the FeedbackGenerator code to a `generateFeedback()` function.
So whenever we need to use it, we would do it by simply using 
```
if #available(iOS 10.0, *) {
      self.myButton.generateFeedback()
    }
```